tensorflow/lite/micro/kernels/xtensa_hifimini/fully_connected.cc

```diff
@@ -26,6 +47,7 @@ limitations under the License.
 #include "tensorflow/lite/kernels/kernel_util.h"
 #include "tensorflow/lite/micro/kernels/xtensa_hifimini/fixedpoint_utils.h"
 
+#include "xtensa_tf_micro_common.h"
```

Comment:
Here and below, this should be a full path "tensorflow/lite/kernels/xtensa_hifimini/xtensa_tf_micro_common"

Reply:
updated based on comment